src/web-fls-quiz/Data/DataStorage.cs

```diff
@@ -129,5 +138,29 @@ public bool InsertQuizResult(QuizResult quizResult)
 
             return true;
         }
+
+        public async Task<StandardImage> GetStandardImage(int id)
+        {
+            var collection = await GetStandardImagesCollection();
+
+            if (collection == null)
+                return null;
+
+            return collection
+                .AsQueryable()
+                .Where(x => x.Id == id)
+                .FirstOrDefault();
+        }
+
+        public async Task<int?> GetStandardImagesNumber()
+        {
+            var collection = await GetStandardImagesCollection();
```

Comment:
Might not be the best solution. I'd rather that we read all picture ID's and then pick one randomly.